README.md

```diff
@@ -8,10 +8,32 @@
 $ docker build -t r2rml .
 ```
 
-### The run the R2RML processor, execute the following command:
+### To run R2RML on Drill for data2services project
+
+A default config.properties file is provided for the data2services project.
+
+This file can be find in the root of the container: `/config.properties`
+
+```shell
+docker run -it --rm --link drill:drill -v /data/my_dataset:/data r2rml /config.properties
+```
+
+* Content of the `config.properties` file:
+
+```properties
+connectionURL = jdbc:drill:drillbit=drill:31010
+mappingFile = /data/mapping.ttl
+outputFile = /data/rdf_output.nq
+format = NQUADS
```

Comment:
we don't need a default config file inside the container. what is the rationale behind this?

Reply:
Having an out of the box example for drill usage